code/game/objects/items/devices/scanners.dm

```diff
@@ -410,8 +410,8 @@ GENE SCANNER
 		if(cyberimp_detect)
 			render_list += "<span class='notice ml-1'>Detected cybernetic modifications:</span>\n"
 			render_list += "<span class='notice ml-2'>[cyberimp_detect]</span>\n"
-
-	to_chat(user, jointext(render_list, ""), trailing_newline = FALSE) // we handled the last <br> so we don't need handholding
+// we handled the last <br> so we don't need handholding
+	to_chat(user, jointext(render_list, ""), trailing_newline = FALSE, type = MESSAGE_TYPE_INFO)
```

Comment:
The comment should prob be on the same indentation level as the to_chat call, since that's what it describes